eng/ProjectReferences.props

```diff
@@ -10,6 +10,7 @@
     <ProjectReferenceProvider Include="Microsoft.Extensions.Caching.SqlServer" ProjectPath="$(RepoRoot)src\Caching\SqlServer\src\Microsoft.Extensions.Caching.SqlServer.csproj" RefProjectPath="$(RepoRoot)src\Caching\SqlServer\ref\Microsoft.Extensions.Caching.SqlServer.csproj" />
     <ProjectReferenceProvider Include="Microsoft.Extensions.Caching.StackExchangeRedis" ProjectPath="$(RepoRoot)src\Caching\StackExchangeRedis\src\Microsoft.Extensions.Caching.StackExchangeRedis.csproj" RefProjectPath="$(RepoRoot)src\Caching\StackExchangeRedis\ref\Microsoft.Extensions.Caching.StackExchangeRedis.csproj" />
     <ProjectReferenceProvider Include="Microsoft.Extensions.Configuration.Abstractions" ProjectPath="$(RepoRoot)src\Configuration\Config.Abstractions\src\Microsoft.Extensions.Configuration.Abstractions.csproj" RefProjectPath="$(RepoRoot)src\Configuration\Config.Abstractions\ref\Microsoft.Extensions.Configuration.Abstractions.csproj" />
+    <ProjectReferenceProvider Include="Microsoft.Extensions.Configuration.Azure.KeyVault.Secrets" ProjectPath="$(RepoRoot)src\Configuration\Config.Azure.KeyVault.Secrets\src\Microsoft.Extensions.Configuration.Azure.KeyVault.Secrets.csproj" RefProjectPath="$(RepoRoot)src\Configuration\Config.Azure.KeyVault.Secrets\ref\Microsoft.Extensions.Configuration.Azure.KeyVault.Secrets.csproj" />
```

Comment:
why does this still have a refprojectpath? Did you regenerate the project list after setting HasReferenceAssembly to false?